src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -44,20 +48,36 @@ class BaseExpensiTextInput extends Component {
     onFocus() {
         if (this.props.onFocus) { this.props.onFocus(); }
         this.setState({isFocused: true});
-        if (this.props.value.length === 0) {
+        this.activateLabel();
+    }
+
+    onBlur() {
+        if (this.props.onBlur) { this.props.onBlur(); }
+        this.setState({isFocused: false});
+        this.deactivateLabel();
+    }
+
+    setValue(v) {
```

Comment:
Add [jsdoc](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs) for param type

Reply:
Done